config/zope-product/tox.ini.j2

```diff
@@ -0,0 +1,76 @@
+{% set with_coverage = True %}
+{% include 'tox-envlist.j2' %}
+
+[testenv]
+skip_install = true
+deps =
+    zc.buildout
+commands_pre =
+{% if testenv_commands_pre %}
+  {% for line in testenv_commands_pre %}
+    %(line)s
+  {% endfor %}
+{% else %}
+    py27,py35: {envbindir}/buildout -nc {toxinidir}/buildout4.cfg buildout:directory={envdir} buildout:develop={toxinidir} install test
+    !py27-!py35: {envbindir}/buildout -nc {toxinidir}/buildout.cfg buildout:directory={envdir} buildout:develop={toxinidir} install test
+{% endif %}
+commands =
+{% if testenv_commands %}
+  {% for line in testenv_commands %}
+    %(line)s
+  {% endfor %}
+{% else %}
+    {envbindir}/test -v []
+{% endif %}
+
+[testenv:lint]
+basepython = python3
+allowlist_externals =
+    mkdir
+commands_pre =
+    mkdir -p {toxinidir}/parts/flake8
+commands =
+    - isort --check-only --diff {toxinidir}/src setup.py
+    - flake8 --format=html src setup.py
```

Comment:
Do we need the `-` here in front of the command? What about more or less `{toxinidir}`

Reply:
It makes that the exit code of the command is ignored. so we see any problems on screen instead only in the HTML file.

Reply:
I'm guessing most people will use `-p auto` or `-p all` - after finding out about that option and how much time it saves it's become my default.

Reply:
@dataflake So you'd opt to remove the `-` in front of the `isort` call so failures show up even when running the tests in parallel?